applications/live-chat.md

```diff
@@ -0,0 +1,15 @@
+---
+description: Machine translation for live chats
+---
+
+# Live Chat
+
+Machine translation for **live chat** is used to translate messages between two users who do not speak a common language.
+
+Live chat messages are a type of **user-generated content**. Live chat is challenging for machine-translation because it is noisy and context-dependent.
```

Comment:
Here and throughout, when mentioning topics for which there is an article (or should be an article), hyperlink, not bold.

(Wiki style)